pycaret/internal/display/__init__.py

```diff
@@ -1,5 +1,4 @@
-from pycaret.utils import enable_colab  # Imported in other modules - do not remove
-
+from ...utils.utils import enable_colab
```

Comment:
should we specify fill path here?

Reply:
The full path here is `from pycaret.internal.display.display import CommonDisplay, DummyDisplay`. Same "issue" as before. There is a dir called display with a file called display